Clang-tidy fixes

QtChecker/include/QtBinding.hpp:6:9: warning: variadic macro 'Q_OBJECT_BINDABLE_PROPERTY' used; consider using a 'constexpr' variadic template function [cppcoreguidelines-macro-usage]
#define Q_OBJECT_BINDABLE_PROPERTY(CLASS, TYPE, NAME, ...) TYPE NAME;

--- a/QtChecker/include/QtBinding.hpp
+++ b/QtChecker/include/QtBinding.hpp
@@ -3,7 +3,7 @@
 #include <QtCore/qglobal.h>
 
 #if QT_VERSION_MAJOR == 5
-#define Q_OBJECT_BINDABLE_PROPERTY(CLASS, TYPE, NAME, ...) TYPE NAME;
+#define Q_OBJECT_BINDABLE_PROPERTY(CLASS, TYPE, NAME, ...) TYPE NAME; // NOLINT(cppcoreguidelines-macro-usage)
 #define BINDABLE(TYPE, NAME, PROPERTY_NAME)
 #else
 #define BINDABLE(TYPE, NAME, PROPERTY_NAME) QBindable<TYPE> NAME() { return &PROPERTY_NAME; }

